matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/model/SessionRealmModule.kt

```diff
@@ -72,7 +72,6 @@ import org.matrix.android.sdk.internal.database.model.threads.ThreadSummaryEntit
             SpaceParentSummaryEntity::class,
             UserPresenceEntity::class,
             ThreadSummaryEntity::class,
-            SyncFilterParamsEntity::class,
```

Comment:
Do we need a database migration because of this change?

Reply:
I've tested upgrading develop build with this branch and there were no problems

Reply:
I added the migration to cleanup the database from the entity added in 45